performant-software/textlab
# Fixes ghost-tabs when rapidly switching leaves in the editor

**What does this do**
closes #183 
Fixes race condition where rapidly switching leaves on the tree caused the editor to open content belonging to previously visited leaves. 

**How to test**
Go Here: https://mel-textlab-pr-190.herokuapp.com/
Open the editor and rapidly switch tabs. Verify only the correct content loads, and especially not "ghost content" from previous leaves.
